source/contracts/reporting/Market.sol

```diff
@@ -680,4 +677,13 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
         }
         return _winningDestination.getUniverse().getParentPayoutDistributionHash();
     }
+
+    // Markets hold the initial fees paid by the creator in ETH and REP, so we dissallow ETH and REP extraction by the controller
+    function getProtectedTokens() internal returns (address[] memory) {
+        address[] memory _protectedTokens = new address[](3);
+        _protectedTokens[0] = address(0);
+        _protectedTokens[1] = reportingWindow.getReputationToken();
+        _protectedTokens[0] = cash;
```

Comment:
`[0]` should be `[2]`.

Reply:
Woops. that suggests tests are passing incorrectly too.

Reply:
We should probably use `address(1)` instead of `address(0)` for the magic token, since `0` is the default if the collection is empty.